### PR TITLE
README edits

### PR DESCRIPTION
**Pull request by amaddalena**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -82,32 +70,27 @@ Installing Elasticsearch
 
 Recommended version **2.4.0**, Moloch versions since 0.15.0 requires at least 2.1.x
 
-1. Prep the ``elasticsearch`` machines by increasing max file descriptors add
-   allowing memory locking. 
-   On CentOS and others this is done by adding the following to bottom of
+1. Prep the ``elasticsearch`` machines by increasing max file descriptors add allowing memory locking.
```
Comment (amaddalena) [inline README.rst]: indeed.

File: README.rst [hunk 2]
```diff
@@ -7,15 +7,9 @@
...
-JSON-formatted session data to be downloaded directly. Simple security is
-implemented by using HTTPS and HTTP digest password support or by using apache
-in front. Moloch is not meant to replace an IDS but instead work along side 
-them to store and index all the network traffic in standard PCAP format, providing 
-fast access.  Moloch is built to be deployed across many systems and can scale to 
-handle tens of gigabits/sec of traffic. 
+Moloch is an open source, large scale, full packet capturing, indexing, and database system. Moloch augments your current security infrastructure to store and index network traffic in standard PCAP format, providing fast, indexed access. An intuitive and simple web interface is provided for PCAP browsing, searching, and exporting. Moloch exposes APIs which allow for PCAP data and JSON formatted session data to be downloaded and consumed directly. A simple web interface is provided for PCAP browsing, searching, analysis and exporting. Moloch stores and exports all packets in standard PCAP format allow you to also use your favorite PCAP ingesting tools, such as wireshark, during your analysis workflow.
```
Comment (31453) [inline README.rst]: Duplicate (kinda) sentences: 
> An intuitive and simple web interface is provided for PCAP browsing, searching, and exporting.
then
> A simple web interface is provided for PCAP browsing, searching, analysis and exporting.

Comment (amaddalena) [inline README.rst]: fixed - removed duplication

**Comment by 31453** [comment]
👍  LGTM

**Pull request merged**